### PR TITLE
Muon prod5 corsika

### PR DESCRIPTION
**Pull request by Voutsi**
Updating corsika cards for muon production in order to be compatible with the Corsika version used currently and also to be able to be accessed by sim_runner.

### CODE REVIEW

**Review by maxnoe — CHANGES_REQUESTED** [review]
I think it would be better to remove the now commented lines instead of keeping them commented.

**Comment by Voutsi** [comment]
Fine by me, if Ruben is ok with that I will remove them

File: config_LST1_muons.cfg [hunk 1]
```diff
@@ -11,106 +11,108 @@
...
-NSHOW   1000                         // number of showers to generate
-#ifdef NSHOW
-NSHOW    $(NSHOW)            // Requires NSHOW environment variable.
-#endif
+NSHOW   1000          // number of showers to generate
+* #ifdef NSHOW
+* NSHOW    $(NSHOW)            // Requires NSHOW environment variable.
+* #endif
 *
 THETAP  0.  0.      // Range of zenith angles (degrees)
```
Comment (maxnoe) [inline config_LST1_muons.cfg]: This is repeated further down

**Comment by rlopezcoto** [comment]
>Fine by me, if Ruben is ok with that I will remove them

sure, sounds good

**Pull request merged**